gneiss/balances.py

```diff
@@ -129,3 +131,163 @@ def _count_matrix(treenode):
             counts[n]['k'] = counts[n.parent]['k'] + counts[n.parent]['r']
             counts[n]['t'] = counts[n.parent]['t']
     return counts, n_tips
+
+
+def default_layout(node):
+    if node.is_leaf():
```

Comment:
Yes.  Because I have no idea how to test it.  

I think this scenario is ok.  ETE layouts are only used to specify the parameters for the TreeStyle.